src/foam/nanos/google/api/sheets/import/GoogleSheetsDataImportServiceImpl.js

```diff
@@ -229,7 +229,8 @@
           Object val = data.get(i).get(columnIndex);
           pasreAndSetValue(x, obj, importConfig.getColumnHeaderPropertyMappings()[j], data.get(i).get(columnIndex));
         }
-        postSetValues(x, obj);
+        if ( ! postSetValues(x, obj) )
+          return new ArrayList<>();
```

Comment:
Will this ArrayList ever be updated? Would an immutable List be sufficient.

Reply:
fixed